h2o-helm/templates/statefulset.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ include "h2o-helm.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+  {{- include "h2o-helm.labels" . | nindent 4 }}
+spec:
+  serviceName: h2o-service
+  podManagementPolicy: "Parallel"
+  replicas: {{ .Values.h2o.nodeCount }}
+  selector:
+    matchLabels:
+  {{- include "h2o-helm.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+    {{- include "h2o-helm.selectorLabels" . | nindent 8 }}
+    spec:
+      securityContext:
+      {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.name }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          command: ["/bin/bash", "-c", "java -XX:+UseContainerSupport -XX:MaxRAMPercentage={{ .Values.h2o.memoryPercentage }} -jar /opt/h2oai/h2o-3/h2o.jar"]
```

Comment:
Can you add optional javaArgs and h2oArgs values?
This way you can pass additional args to Java/H2O without needing to modify the Chart.

Reply:
Removed.

Reply:
I've put in fully customizable command, as discussed. A command needs to be fully overridable in case custom docker image is used. Would introduce additional complexity especially for users if custom args were also present - they'd be confused when and what is actually enabled :wink: